webpack.config.js

```diff
@@ -0,0 +1,2 @@
+// This file is for helping IntelliJ
+module.exports = require("./config/webpack/webpack.dev")
```

Comment:
This should be in the dev webpack config file.

Reply:
No. I stated in the comment, this file is only here to help IntelliJ with module import auto-completion. IntelliJ looks for this file at the root of the project and detects WP configuration.

Reply:
I think you can define it in settings? But sure, I don't mind. But could you clarify with the longer version of the comment like above in the file and also add a comment indicating where the real webpack config is?